deeppavlov/core/agent_v2/state_schema.py

```diff
@@ -13,6 +14,33 @@ def to_dict(self):
                 'user_type': self.user_type,
                 'personality': self.personality}
 
+    @classmethod
+    def get_or_create(cls, *args, **kwargs):
+        """
+        gets or creates an object from init specification
+
+        Args:
+            *args:
+            kwargs:
+
+        Returns: tuple (instance:User, is_created:bool)
+            or raises Exception if Multiple instances are found
```

Comment:
Google style docstrings is accepted in the project. Please make yourself familiar with it:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
There is a special syntax for raised Exceptions in docstrings.

Reply:
fixed